internal/backend/dispatcher/eventsworkflow.go

```diff
@@ -266,82 +249,6 @@ func (d *dispatcher) eventsWorkflow(ctx workflow.Context, input eventsWorkflowIn
 		return nil, err
 	}
 	// Set event to Completed
-	if err = d.createEventRecord(context.Background(), input.EventID, sdktypes.EventStateCompleted); err != nil {
-		return nil, err
-	}
-
+	d.CreateEventRecord(context.Background(), input.EventID, sdktypes.EventStateCompleted)
```

Comment:
why no error handling here?

Reply:
createEventRecord returns nothing. It panics. This is the way it was previously as well, but it was always returning nil